src/sentry/static/sentry/app/views/settings/project/projectKeys/projectKeyCredentials.jsx

```diff
@@ -47,6 +47,25 @@ class ProjectKeyCredentials extends React.Component {
 
     return (
       <React.Fragment>
+        {showDsn && (
+          <Field
+            label={t('DSN (Legacy)')}
```

Comment:
Should we maybe not call it Legacy? Legacy, to me, implies that I shouldn't use it and I'd look for the one that's not legacy anyways.

Reply:
I've updated the wording for the help text maybe it's more clear now.

Also AFAIK we want that people move away from it and it should be a conscious choice if you still want to take it.